### ISSUE TITLE
(Long Running): Upload new cset in backend & test in frontend

### ISSUE THREAD

**Issue by Sigfried** [jhu-bids/TermHub#561]

**Comment by joeflack4** [comment]
It looks like I thought there was an existing issue for this w/ notes. Maybe my notes are in the grouping issue? Here's what's in my worfklowy.

<details><summary>Details</summary>
<p>

original notes (i've written up-to-date notes in the issue)

- test cases
  - new cset version
    - backend test for creating new cset version. and a frontend test that tries to load that
- how?
  - python file to orchestrate
  - have python orchestration: run the backend test by name
  - have python orchestration: run frontend test
    - a. have a single file for this test <-
      - less elegant, but siggie has a solution
        - siggie: i added a comment in the issue about how to read a data file before running a frontend test
      - I can add this file to the .gitignore
    - b. pass the ID to the frontend via CLI
      - a. file
        - it makes a record of the codeset id. maybe saves it to a temp file
      - b. cli
        - idk how to do this in node


</p>
</details>

**Comment by Sigfried** [comment]
Yeah, I thought we had one as well. Hmmm. Anyway, we do really need to build it.
